.github/PULL_REQUEST_TEMPLATE/bug-fix.md

```diff
@@ -0,0 +1,53 @@
+## Description
+<!-- Briefly describe the bug you are trying to fix -->
+
+<!-- Describe your changes in detail -->
+<!-- If it fixes an open issue, please link to the issue here. -->
+
+## Checklist
+<!-- Feel free to remove check-list items aren't relevant to your change -->
+
+- [ ] Is this PR following the [official](https://github.com/arviz-devs/arviz/blob/master/CONTRIBUTING.md#pull-request-checklist) 
+      PR format (described below)?
+- [ ] Have you added the unit tests to prevent issue reoccurance( using [pytest fixture pattern](
+      (https://docs.pytest.org/en/latest/fixture.html#fixture))?
+- [ ] Have you run the unit tests using pytest?
```

Comment:
I am not sure about this, I would probably remove it. There is no need to run them locally, changes can be pushed to see if tests pass in azure/travis.

Reply:
Yes, they have to add unit tests to prevent issue reoccurrence, right?

Reply:
Yes, but there are two points in the checklist about tests. The first is about the PR having tests and the second is about running tests locally, this comment is only about the running tests locally checkpoint.